gladier_tools/posix/asym_encrypt.py

```diff
@@ -0,0 +1,62 @@
+from gladier import GladierBaseTool, generate_flow_definition
+
+
+def asym_encrypt(**data):
+    import os
+    from cryptography.hazmat.primitives.serialization import \
+        load_ssh_public_key
+    from cryptography.hazmat.primitives.asymmetric import padding
+    from cryptography.hazmat.primitives import hashes
+
+    public_file = data['public_key_path']
+    if '~' in public_file:
+        public_file = os.path.expanduser(public_file)
+
+    with open(public_file, 'rb') as pub_file:
+        public_key = load_ssh_public_key(pub_file.read())
+
+    infile = data['asym_encrypt_file']
+    if '~' in infile:
+        infile = os.path.expanduser(infile)
+
+    with open(infile, 'rb') as in_file:
+        message = in_file.read()
+
+    ciphertext = public_key.encrypt(
+        message,
+        padding.OAEP(
+            mgf=padding.MGF1(algorithm=hashes.SHA256()),
+            algorithm=hashes.SHA256(),
+            label=None
+        )
+    )
+
+    with open(infile+'.rsa', 'wb') as outfile:
+        outfile.write(ciphertext)
+
+    return infile+'.rsa'
```

Comment:
I think this more of a nitpick than anything else, I don't really have a good reason for it. But I more typically use format or f-strings for formatting text than string concatenation. I think f-strings/format tend to be a bit more resilient to unexpected types and a bit more readable. 

I'll leave that up to you though if you want to change it or not!

Reply:
Makes sense! Ill change it